src/main/resources/templates/db_scripts/schema.json

```diff
@@ -313,8 +313,22 @@
     },
     {
       "tableName": "instance",
-      "fromModuleVersion": "17.1.0",
+      "fromModuleVersion": "19.2.0",
       "withMetadata": true,
+      "withAuditing": true,
+      "auditingTableName": "audit_instance",
+      "auditingFieldName": "instance",
+      "auditingSnippet": {
+        "delete": {
+          "statement": "jsonb = jsonb_build_object('id', uuid, 'operation', to_jsonb(left(TG_OP, 1)), 'deletedDate', to_jsonb(current_timestamp::text));"
+        },
+        "insert": {
+          "statement" : "return null;"
```

Comment:
For inserts and update the trigger is started by exists early without writing to the audit table. This degrades performance. The trigger should not run at all for inserts and updates.
You may use `"customSnippetPath": "audit-delete-trigger.sql"` in each table:
```
DROP TRIGGER IF EXISTS audit_${table.tableName} ON ${myuniversity}_${mymodule}.${table.tableName} CASCADE;

CREATE TRIGGER audit_${table.tableName} AFTER DELETE ON ${myuniversity}_${mymodule}.${table.tableName}
  FOR EACH ROW EXECUTE PROCEDURE ${myuniversity}_${mymodule}.audit_${table.tableName}_changes();
```